### PR TITLE
Russian translation

### PR DESCRIPTION
**Pull request by rchintsov**
Full Russian translation. Based on the original file (follows instructions from CONTRIBUTING.md).
Contains a few TODOs for future work.

### CODE REVIEW

**Comment by forrestguice** [comment]
@rchintsov I'll merge this later this week. 

If you need to make additional changes, you can continue making commits (and push them), and they will show up here automatically. That should trigger the CI to produce another debug apk.

**Comment by rchintsov** [comment]
@forrestguice Ok, thanks. I'll commit fixes within a hour.

**Comment by rchintsov** [comment]
@forrestguice All job is done. If there are any changes, I think they'll be very small.
Please, take a look at TODO notes, I can't solve most of it without your help.

File: app/src/main/res/values-ru/strings.xml [hunk 1]
```diff
@@ -1531,109 +1537,110 @@
...
-        See <b>Intents and Intent Filters</b> <small>(https://developer.android.com/guide/components/intents-filters)</small>.
+        См. <b>Intents and Intent Filters</b> по адресу<br/><a href="https://developer.android.com/guide/components/intents-filters">developer.android.com/guide/components/intents-filters</a>.
         <br/><br/>
-        All fields are optional (values may be left empty for an <em>implicit intent</em>).
+        Все поля необязательны (значения могут быть оставлены пустыми для использования значений по умолчанию).
         <br/>
         <br/>
-        <b>Action:</b> An action string.<br/><small>e.g. <em>android.intent.action.VIEW</em></small>
+        <b>Действие:</b> Строка действия. <br/><small>напр. <em>android.intent.action.VIEW</em></small>
         <br/>
```
Comment (forrestguice) [inline app/src/main/res/values-ru/strings.xml:1583]: This should probably be "Action:</b> Строка действия." to match the UI (Android specific term).

**Comment by forrestguice** [comment]
This looks good. 👍 It does pass for a native app, congrats on the great work.

It is very complete, so I tried fixing anything I saw that wasn't translated - a handful of dialogs with untranslated "cancel" buttons. I also fixed those plurals. If you pull this branch they should now be marked `TODO`. 

I'll give some thought to the note area (to avoid scrolling). I also noted that the `_summary` strings were broken as if they are hardcoded to only display `%s`. This is something I'll have to fix.

It also seems that the 12-hr format (translated) is broken for older devices. My older phone (api19) displays "до полудня" and "после полудня" (way too long), while the latest emulator displays "AM" and "PM". I'm not sure there's anything I can do to fix it - its just the way `DateTimeFormatter` chose to format pattern `a`. I think seeing this will be uncommon though - most Russian speakers will be using 24hr time?

File: app/src/main/res/values-ru/strings.xml (hunk 1 above)
Comment (rchintsov) [reply app/src/main/res/values-ru/strings.xml:1583]: Ok, now it's clear. Thanks.

**Comment by rchintsov** [comment]
@forrestguice 
> congrats on the great work.

Thanks! And you on expanding your audience!

>  It does pass for a native app

I imagine nativeness not as fullness of translation, but as nativeness of using the language. Most of translations maked by direct translation of the English words. This looks very rude, I tried to avoid this.
IMHO, user should think, that the app author built the UI followed by nuances of the language (for Russian it's true). If initially it wasn't like that, it's quite difficult to achieve that feel just by translation. I tried to do this and I think I succeeded, so the result looks for me a little bit incredible.

> a handful of dialogs with untranslated "cancel" buttons

I didn't pay attention for untranslated "cancel" buttons, but I remember UI theme names with degrees of transparency in brackets, that I couldn't translate. It may be more important for users, since we are already used to seeing buttons in English in other apps. 

Thanks for your attention to scrolling and other. I'm waiting for the opportunity to write more precise wordings of twilights.
BTW, this can be done more beautifully by dividing events into continuing (like twilight) and one-time (like noon). Then we can add `<string name="until_start">...</string>` to handle continuing events separately.

> I think seeing this will be uncommon though - most Russian speakers will be using 24hr time?

I can't answer for all Russian speakers, but I've never seen a person using 12hr time together with the Russian interface. So, don't worry about that. Also, we never think about it, just using the default 24h preset for the Russian locale.

P.S.: I just remembered my big surprise when I first encountered 12h time in the school English classes. It was very unusual :)

**Comment by forrestguice** [comment]
btw, its also possible to translate the F-Droid listing. I've added the required files (`fastlane/metadata/android/ru/full_description.txt` and `short_description.txt`). Those 'place' plurals are also available for translation.

We can include those in this pull, or wait for the next patch (most likely ~2 weeks), or the patch after that, etc.
I am going to merge this and tag v0.13.5 tomorrow. After that its usually around a week before it shows up in F-Droid.

**Comment by rchintsov** [comment]
@forrestguice Ok, I'm in. But I'll do this tomorrow (if I got this right, tomorrow for me and for you - [are different things](https://time.is/en/compare/0000_4_Dec_2020_in_Prescott_Valley/MSK)). Anyway, the main point is that we'll be in time.

**Comment by rchintsov** [comment]
@forrestguice I experimented a bit with the look of the permission list and changed the link to Suntimes Calendars. This is because the F-Droid site handle links to itself in its own way (with `<tt>` tag). Let's see what happens.

**Comment by forrestguice** [comment]
I saw that broken link a few days ago and wondered how to fix it. I was unaware of `<tt>`. Is this it documented somewhere?

**Comment by rchintsov** [comment]
@forrestguice I don't know. I just looked to HTML and saw that the `<a>` tag divided in two by `<tt>`.

**Comment by rchintsov** [comment]
@forrestguice Look: now it's displaying good (screen from F-Droid). It remains to remove `<small>` tags for better look. Note that I removed `=` sign before `<=`  next to API. It's unnecessary there. 

изображение

**Comment by forrestguice** [comment]
Its looks good. Using the translated description to fine tune this is a good idea. Please experiment more if you'd like, and later I will apply changes to the other files.

I'm in favor of removing HTML formatting. It was an experiment, but it seems to look better without.

**Pull request merged**